api-docs/leads.md

```diff
@@ -0,0 +1,332 @@
+# Leads
+
+The leads document direct encounters with leads.
+
+Leads are potiental clients.
+
+Here is the list of the valid lead types:
+
+- `leadCreated`,
+- `leadUpdated`,
+- `leadDeleted`
+
+## Get a single Lead
+
+### Request
+
+```http
+GET /api/leads/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "dateOfSignup": "2019-09-17",
+  "leadStatus": "pending",
+  "firstName": "Joel",
+  "lastName": "Denning",
+  "fullName": "Joel Denning",
+  "phone": "555-555-5555",
+  "smsConsent": true,
+  "zip": "84115",
+  "age": 25,
```

Comment:
Age changes, birthday doesn't. We should change to birthday (or birth year) instead of age in the API.

Reply:
The current form only collects the age. I think they grab the birthday during intake only.

Reply:
Hmm but putting ages into a database doesn’t make a ton of sense, since time passes and it becomes stale.

The UI could prompt for age and then make a guess for the birth year and send that up to the API? What do you think?